scripts/deploy-backend.sh

```diff
@@ -112,6 +112,9 @@ set_alloy_envs() {
 }
 
 add_service_bindings() {
+    yq eval -i ".applications[0].services[3] = \"tdp-redis-${env}\"" ./tdrs-backend/manifest.buildpack.yml
+    yq eval -i ".applications[0].services[3] = \"tdp-redis-${env}\"" ./tdrs-backend/manifest.celery.yml
+
     if [ "$CGAPPNAME_BACKEND" = "tdp-backend-develop" ]; then
       # TODO: this is technical debt, we should either make staging mimic tanf-dev
       #       or make unique services for all apps but we have a services limit
```

Comment:
seems like we _have_ `-staging` services (bound to the `-staging` applications) - can we just get rid of this? or is there more work to do with databases/s3/etc?